elements/pf-button/pf-button.css

```diff
@@ -570,7 +570,7 @@ button:hover {
 :host([block]) button {
   display: flex;
   width: 100%;
-  justify-content: center;
+  justify-content: var(--pf-c-button--block--JustifyContent, center);
```

Comment:
can we do this with a ::part instead i wonder? Would that be better?

Reply:
I think this would be a good case for `::part`.  @eyevana if we find ourselves adding CSS variables for individual properties on a single element in the shadowdom then I would propose we add a shadow part.

pf-button/BaseButton.ts
```js
  override render() {
      <button part="button" ...> ... </button>
    `;
  }
```

styles.css
```css
pf-button[block]::part(button) {
  justify-content: center;
  margin-block-start: 3.14159px;
}
```

Reply:
Gotcha! I'll update my changes now and add a part instead. Thanks for the feedback.

Reply:
ping =) sorry to be annoying, I just need this little itty bitty change for the work I'm doing in cp.

Reply:
not annoying, thanks for keeping reviewers focused.